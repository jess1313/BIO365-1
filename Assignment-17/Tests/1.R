suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("1.R"))))

set.seed(0)

print(standardize(rnorm(10)))
print(standardize(rnorm(100)))
print(standardize(rnorm(1000)))