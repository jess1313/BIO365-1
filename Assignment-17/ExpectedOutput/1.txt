[1] 4.581838e-17 1.000000e+00
[1] -1.807257e-17  1.000000e+00
[1] -2.248782e-17  1.000000e+00
